src/Engine/Manager/PhysicsManager.cpp

```diff
@@ -95,42 +102,51 @@ void PhysicsManager::Update(float deltaTime) {
 }
 
 void PhysicsManager::UpdateEntityTransforms() {
-    std::vector<Component::Physics*> physicsObjects = physicsComponents.GetAll();
-    for (Component::Physics* physicsComp : physicsObjects) {
-        if (physicsComp->IsKilled() || !physicsComp->entity->enabled)
+    for (auto rigidBodyComp : rigidBodyComponents.GetAll()) {
+        if (rigidBodyComp->IsKilled() || !rigidBodyComp->entity->enabled)
             continue;
 
-        Entity* entity = physicsComp->entity;
+        Entity* entity = rigidBodyComp->entity;
 
-        auto trans = physicsComp->GetRigidBody().GetRigidBody()->getWorldTransform();
+        auto trans = rigidBodyComp->GetBulletRigidBody()->getWorldTransform();
         entity->position = Physics::btToGlm(trans.getOrigin());
         entity->rotation = glm::eulerAngles(Physics::btToGlm(trans.getRotation()));
     }
 }
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'MakeRigidBody' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)